metaflow/plugins/aws/batch/batch_cli.py

```diff
@@ -141,8 +141,9 @@ def kill(ctx, run_id, user, my_runs):
 @click.option("--max-swap", help="Max Swap requirement for AWS Batch.")
 @click.option("--swappiness", help="Swappiness requirement for AWS Batch.")
 # TODO: Maybe remove it altogether since it's not used here
-@click.option("--ubf-context", default=None, type=click.Choice([None]))
+@click.option("--ubf-context", default=None, type=click.Choice([None, "ubf_control"]))
```

Comment:
We have other "--ubf-context" options but they are not always a choice between None and a string. In the main CLI the default is the string "none" for example. Maybe we could make them all consistent to avoid any potential issues/pitfalls?

Reply:
The solution in the main cli is weird, but it then has:

```
    if ubf_context == "none":
        ubf_context = None
```
Since this CLI is always called by the metaflow itself (never by the user), I don't see why we should support "none" string input. And if that gets passed from somewhere, this will fail gracefully.

And as this PR did not actually change this (the None was the sole argument option before), I suggest leaving this as is. Follow-up PR to clean up the main CLI can be done if wanted.

Reply:
Sounds ok. I think it was due to something click related which wasn't happy with multiple types (or something). As long as the result is consistent, should be fine.